tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -408,6 +408,13 @@ struct PersistentRnnPlanDeleter {
     CHECK_CUDNN_OK(cudnnDestroyPersistentRNNPlan(plan));
   }
 };
+#if CUDNN_VERSION >= 7601
```

Comment:
Can we not change stream_executor and instead call cudnn directly from the kernel?

Reply:
But I think all the cudnn calls are inside this cuda_dnn.cc. I haven't seen any example to call the cudnn directly from the kernel.